backends/arm/test/tester/arm_tester.py

```diff
@@ -34,53 +34,65 @@
 from executorch.backends.xnnpack.test.tester import Tester
 from executorch.exir import EdgeCompileConfig
 from executorch.exir.backend.compile_spec_schema import CompileSpec
+from executorch.exir.backend.partitioner import Partitioner
 from executorch.exir.lowered_backend_module import LoweredBackendModule
 from torch.fx import Graph
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 
+def _dump_lowered_modules_artifact(
```

Comment:
Just refactoring I assume? Try to keep these in a different commit (same PR is fine).

Reply:
Mainly because I needed the new `class ToEdgeTransformAndLower(tester.ToEdgeTransformAndLower)` and so I extracted this so that it can be shared by `class Partition(tester.Partition)`.

I can eventually create a separate commit for the whole `backends/arm/test/tester/arm_tester.py` changes if you prefer as it's effectively not directly related to the upsample_nearest2d. Just some additions to the tester to properly test the op.

Reply:
All good. We can do different commits next time :)